core/src/integrationTest/kotlin/com/malinskiy/marathon/scenario/CacheScenarios.kt

```diff
@@ -34,12 +33,8 @@ class CacheScenarios : Spek(
             container.stop()
         }
 
-        afterEachTest {
-            stopKoin()
```

Comment:
This pattern is used in several places. Why does it work there or do they need to be updated?

Reply:
`group("the second execution of the test")` is the only place where we run marathon 2 times during the test. In every other test we run marathon only once and stop Koin after that. But with two runs it's not working as we need to stop it after each run.